packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -84,10 +88,47 @@ export interface ITreeConfigurationOptions {
 	 * bit slower.
 	 */
 	enableSchemaValidation?: boolean;
+
+	/**
+	 * If `true`, constructing the configuration will error if any valid input data could parse ambiguously
+	 * (type of the node could fail to be inferred from content, thus requiring use of {@link Unhydrated} nodes to disambiguate).
```

Comment:
Using unhydrated nodes to disambiguate would be a great thing to put in an `@example` section in this doc, and having an example would help the reader better understand the idea overall.

Reply:
done.